core/kt-osrd-sim-infra/src/main/kotlin/fr/sncf/osrd/sim_infra/utils/EnvelopeTrainPathUtils.kt

```diff
@@ -0,0 +1,31 @@
+package fr.sncf.osrd.sim_infra.utils
+
+import fr.sncf.osrd.envelope_sim.electrification.Electrification
+import fr.sncf.osrd.envelope_sim.electrification.Electrified
+import fr.sncf.osrd.envelope_sim.electrification.Neutral
+import fr.sncf.osrd.envelope_sim.electrification.NonElectrified
+import fr.sncf.osrd.sim_infra.api.Path
+import fr.sncf.osrd.sim_infra.impl.DeadSection
+import fr.sncf.osrd.utils.DistanceRangeMap
+import fr.sncf.osrd.utils.DistanceRangeMapImpl
+import fr.sncf.osrd.utils.units.Distance
+
+
+/**
+ * Builds the ElectrificationMap
+ */
+fun buildElectrificationMap(path: Path): DistanceRangeMap<Electrification> {
```

Comment:
If this is only used in `EnvelopeTrainPath`, it can be a private function there. Unless it's eventually meant to be used at other places

Reply:
It facilitates the creation of kotlin objects to be used in java later